snmp/tests/compose/data/netgear-switch.snmprec

```diff
@@ -0,0 +1,1480 @@
+1.3.6.1.2.1.1.1.0|4|netgear-switch Device Description
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.4526.100.1.28
+1.3.6.1.2.1.1.5.0|4|netgear-switch.device.name
+1.3.6.1.2.1.2.1.0|2|4
```

Comment:
the check is failing because we have this in the snmprec but it is not asserted in the tests
since this profile extends `netgear.yaml` and that profile already tests the `_generic-if.yaml` extend then you don't have to have it again
removing this should fix it
or if you want to keep it you need to add a `assert_generic_if` to the tests